packages/video_player/video_player/lib/video_player.dart

```diff
@@ -622,14 +639,43 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     );
   }
 
+  /// Get all available embedded subtitle of video.
+  ///
+  /// This is useful for case video contains embedded subtitle like Hls. Items
+  /// of response can use by [setEmbeddedSubtitles] to select prepare subtitle.
```

Comment:
Get all available embedded subtitles of the video.

This is useful for video formats containing embedded subtitles like Hls. The response items can be used with [setEmbeddedSubtitles] to select and prepare a subtitle.